src/containers/Validators/index.js

```diff
@@ -67,9 +67,15 @@ class Validator extends Component {
   fetchData = () => {
     const { match } = this.props
     const { identifier = '' } = match.params
-    axios.get(`/api/v1/validator_report/${identifier}`).then((resp) => {
-      this.setState({ reports: resp.data })
-    })
+    axios
+      .get(`${process.env.REACT_APP_DATA_URL}/validator/${identifier}/reports`)
+      .then((resp) => resp.data.reports)
```

Comment:
this is an extra then.

Reply:
Is there something wrong with doing it this way? I find it's a little cleaner to move some of the logic to a `then`.